luaa.c

```diff
@@ -254,15 +254,16 @@ luaA_restart(lua_State *L)
 
 /** Send a signal to a process identified by its process id. See
  * `awesome.unix_signal` for a list of signals.
- * @tparam integer pid Process identifier
+ * @tparam integer pid Process identifier.  0 and negative values have special
+ *   meaning.  See `man 3 kill`.
```

Comment:
Rather 2. ;)

Reply:
I have it as `KILL(3P)` here (also), which I found more appropriate (given its more descriptive information IIRC).